configure.in

```diff
@@ -36,6 +36,28 @@ AC_CHECK_FUNCS([getopt_long getline strtof])
 AC_FUNC_SNPRINTF
 AC_FUNC_SCANF_CAN_MALLOC
 
+AC_MSG_CHECKING(--enable-libarchive argument)
```

Comment:
Okay, I will change the default and instead add a `--disable-libarchive` option to `./configure`. My initial assumption was, that you'd not like the additional dependency because libarchive is quite heavy drawing in all kinds of compression libraries and even libxml, which is quite a drastic change for a tool like genext2fs which didn't have any library dependencies before.

Reply:
Oh effectively, that looks terrible.
I know I already wrote some tar parsing code once, it's not that complex. Maybe one day I'll do it again :)